doc/modules/calibration.rst

```diff
@@ -74,10 +74,12 @@ by showing the number of samples in each predicted probability bin.
 
 .. currentmodule:: sklearn.linear_model
 
-:class:`LogisticRegression` returns well calibrated predictions by default as it has a
+:class:`LogisticRegression` is more likely to return well calibrated predictions by default as it has a
 canonical link function for its loss, i.e. the logit-link for the :ref:`log_loss`.
-This leads to the so-called **balance property**, see [8]_ and
-:ref:`Logistic_regression`.
+This leads to the so-called **balance property**, see [8]_ and :ref:`Logistic_regression`.
```

Comment:
```suggestion
In the unpenalized case, this leads to the so-called **balance property**, see [8]_ and :ref:`Logistic_regression`.
```

Reply:
Sure, we can do this.

If I were to actually rethink this text, I would remove the reference to the "balance property", which is too technical for most readers. I would prefer focusing on the big pictures, which is that proper scoring rules as losses are more likely to return calibrated probabilities, but they do need appropriate choice of model.